src/Config/Loader/PhpFileLoader.php

```diff
@@ -52,7 +52,29 @@ public function load($file, $type = null)
             $code = substr($code, 0, -2);
         }
 
-        return rtrim($code)."\n";
+        // declare() statements
+        $lines = explode("\n", $code);
+        $codeNew = '';
+        $processed = false;
+
+        foreach ($lines as $line) {
+            if ('' === $line || $processed) {
```

Comment:
Just tried it out and it indeed works...
```php
declare(strict_types = 1, ticks = 1, encoding='UTF-8');
```

Funny is, that the following only raises a warning, maybe we need to handle all declares then?
```php
declare(foo='bar');
```

Another thing some people might want to do:
```php
<?php declare(strict_types = 1, ticks = 1, encoding='UTF-8');

/***
 * file doc goes here
 */
```

This gets more and more a bottomless pit... :grimacing: